readme.md

```diff
@@ -53,7 +54,7 @@ body{
 }
 ```
 
-> Your answer...
+> That's improper syntax bud. It should be background-color: red; or background = red; is even better in my opinion
 
```

Comment:
Yes, `background-color: red` would work. But not `background = red`. Can't use `=` when setting properties in CSS.